firmware/robot2015/src-ctrl/modules/controller/controller.hpp

```diff
@@ -0,0 +1,5 @@
+
+#include "robot-devices.hpp"
+
+
+void Task_Controller(void const*);
```

Comment:
What does this function do?  It looks like it's defined in ControllerTaskThread.cpp, so maybe we can drop the empty controller.cpp.